docs/api/advanced/artifacts.md

```diff
@@ -0,0 +1,176 @@
+---
+outline: deep
+title: Test Artifacts
+---
+
+# Test Artifacts <Badge type="danger">advanced</Badge> <Badge type="warning">experimental</Badge>
```

Comment:
We have `<Experimental />` component now 😄 I think we should also add `Advanced`

Reply:
We should also have a version here. I wonder if experimental features should have a `warning` color in the `Version` component though 🤔